lighthouse-core/formatters/partials/speedline.css

```diff
@@ -0,0 +1,3 @@
+.speedline-measures {
```

Comment:
looks like you can just no include a css file for it (or leave an empty one so there's a place for future css?)